packages/loader/container-utils/package.json

```diff
@@ -92,6 +92,26 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {
```

Comment:
This is interesting. I would have expected these to remain compatible, since the only things we moved were interfaces these classes extended. Might be worth re-evaluating if these are needed after any other changes you're making have settled.

Reply:
@ghoshkaj Would you mind trying to refresh the typetests and see if these changes are actually needed? It doesn't seem like they should be. Structurally, we haven't changed anything in these.